connector_prestashop_catalog_manager/models/product_template/common.py

```diff
@@ -54,23 +55,22 @@ class PrestashopProductTemplateListener(Component):
     _apply_on = 'prestashop.product.template'
 
     @skip_if(lambda self, record, **kwargs: self.no_connector_export(record))
-    @skip_if(lambda self, record, **kwargs: self.need_to_export(record, **kwargs))
     def on_record_create(self, record, fields=None):
         """ Called when a record is created """
         record.with_delay().export_record(fields=fields)
 
     @skip_if(lambda self, record, **kwargs: self.no_connector_export(record))
-    @skip_if(lambda self, record, **kwargs: self.need_to_export(record, **kwargs))
+    @skip_if(lambda self, record, **kwargs: self.need_to_export(
+        record, **kwargs))
     def on_record_write(self, record, fields=None):
         """ Called when a record is written """
         record.with_delay().export_record(fields=fields)
         if 'minimal_quantity' in fields:
```

Comment:
While changing `minimal_quantity` it brings recursion error:

```
File "/home/odoo/projec/addons_external/connector/producer.py", line 37, in write
  result = super(Base, self).write(vals)
File "/home/odoo/projec/addons_external/component_event/models/base.py", line 101, in write
  result = super(Base, self).write(vals)
File "/home/odoo/.venv-odoo/lib/python2.7/site-packages/odoo/models.py", line 3568, in write
  self._write(old_vals)
File "/home/odoo/.venv-odoo/lib/python2.7/site-packages/odoo/models.py", line 3704, in _write
  field.write(self.with_context(rel_context), vals[name])
File "/home/odoo/.venv-odoo/lib/python2.7/site-packages/odoo/fields.py", line 2228, in write
  comodel.browse(act[1]).write(act[2])
File "/home/odoo/projec/addons_external/connector/producer.py", line 37, in write
  result = super(Base, self).write(vals)
File "/home/odoo/projec/addons_external/component_event/models/base.py", line 104, in write
  self._event('on_record_write').notify(record, fields=fields)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 190, in notify
  event(*args, **kwargs)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 160, in func_wrapper
  return func(*args, **kwargs)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 160, in func_wrapper
  return func(*args, **kwargs)
File "/home/odoo/projec/addons_external/connector_prestashop_catalog_manager/models/product_template/common.py", line 72, in on_record_write
  'minimal_quantity': binding.minimal_quantity
File "/home/odoo/projec/addons_external/connector/producer.py", line 37, in write
  result = super(Base, self).write(vals)
File "/home/odoo/projec/addons_external/component_event/models/base.py", line 104, in write
  self._event('on_record_write').notify(record, fields=fields)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 190, in notify
  event(*args, **kwargs)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 160, in func_wrapper
  return func(*args, **kwargs)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 160, in func_wrapper
  return func(*args, **kwargs)
File "/home/odoo/projec/addons_external/connector_prestashop_catalog_manager/models/product_template/common.py", line 72, in on_record_write
  'minimal_quantity': binding.minimal_quantity
File "/home/odoo/projec/addons_external/connector/producer.py", line 37, in write
  result = super(Base, self).write(vals)
File "/home/odoo/projec/addons_external/component_event/models/base.py", line 104, in write
  self._event('on_record_write').notify(record, fields=fields)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 190, in notify
  event(*args, **kwargs)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 160, in func_wrapper
  return func(*args, **kwargs)
File "/home/odoo/projec/addons_external/component_event/components/event.py", line 160, in func_wrapper
  return func(*args, **kwargs)
File "/home/odoo/projec/addons_external/connector_prestashop_catalog_manager/models/product_template/common.py", line 72, in on_record_write
....
```

Reply:
It has been fixed, thanks @paulius-sladkevicius !

Reply:
@FFernandez-PlanetaTIC thanks. We do heavy testing now, so happy to contribute at least with issue.